packages/types/package.json

```diff
@@ -21,17 +21,15 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run build",
-    "build": "tsc -p tsconfig.build.json",
-    "dev": "tsc -p tsconfig.build.json --watch",
+    "build": "tsc -p tsconfig.json",
+    "dev": "npm run build -- --watch",
```

Comment:
Thanks for reviewing @panteliselef!
I'm going to revert the change but could you please share a bit more context as to why you find this confusing? It'd be great since we're about to revamp the build/release process - cleaning up the scripts fits in this process nicely :)

Reply:
Done.

Reply:
@panteliselef interested in hearing your thoughts about this as Id like to think about a potential refactor